### ISSUE TITLE
Translations not updating with unstable_revalidate

### ISSUE THREAD

**Issue by mbpictures** [vinissimus/next-translate#803]
Hey!
I've overridden the "loadLocaleFrom" Function in the config to load translations from the database. I also disabled webpack loader and added the ```await loadNamespaces({ locale, pathname: '/' })``` props to all pages with the appWithI18n for the _app. However, every time I update the translations in the db and trigger a rebuild with ```unstable_revalidate```, the translations aren't updated.

Thanks for your help and best wishes!

**Comment by aralroca** [comment]
How is your [`loadLocaleFrom`](https://github.com/vinissimus/next-translate#create-your-namespaces-files) function? The default one use a dynamic import 

https://github.com/vinissimus/next-translate/blob/1.3.5/src/plugin/utils.ts#L6-L7

```js
(locale, namespace) => import(`@next-translate-root/locales/${locale}/${namespace}`).then(m => m.default)
```

I dunno how `revalidate` works, but maybe is not updating the webpack chunk...

**Comment by mbpictures** [comment]
This is my loadLocaleFrom function:
```JavaScript
  async (lang, ns) => {
      const prisma = new PrismaClient();
      let result = (await import(`./locale/${DEFAULT_LANG}/${ns}.json`)).default;
      try {
          result = {...result , ...(await import(`./locale/${lang}/${ns}.json`)).default};
      } catch (e) {
          throw e;
      }
      try {
          const translations = await prisma.translation.findMany({
              where: {
                  namespace: ns
              }
          });
          const db = translations
              .filter((translation) => translation.translations[lang])
              .reduce((result, translation) => {
                  return { ...result, [translation.key]: translation.translations[lang]};
              }, {});
          result = { ...result, ...db }; // override locales by database
      } catch (e) {
          throw e
      } finally {
          prisma.$disconnect();
      }
      return result;
  }
```

I thought using the appWithI18n HOC completely removes the webpack loader?

**Comment by aralroca** [comment]
Do you know if the problem is that the`loadLocaleFrom` is not called after the `unstable_revalidate`?

> I thought using the appWithI18n HOC completely removes the webpack loader?

You also need [`loader: false`](https://github.com/vinissimus/next-translate/blob/master/examples/without-loader/i18n.js#L4) in `i18n.json`.

**Comment by mbpictures** [comment]
Yes, the ```loadLocaleFrom``` is called with ```unstable_revalidate```.

And I set ```loader: false``` in ```i18n.json```, sorry for not mentioning it earlier.

**Comment by Vylda** [comment]
Well, it doesn't work with default loader in my project too.

**Comment by aralroca** [comment]
I'm not sure how we can solve the revalidate problem from our side, it's something internal to Next.js core. Any idea feel free to PR
